examples/statistics.js

```diff
@@ -0,0 +1,12 @@
+const mineflayer = require('mineflayer')
+
+const bot = mineflayer.createBot({
```

Comment:
this isn't the format of an example, I'd suggest looking at other examples to see how they make the bot object

Reply:
Done